### PR TITLE
General tidy up of things

### PR DESCRIPTION
**Pull request by timheap**
Remove some old commented out code, add some more linting (I like isort), make some tests a bit clearer.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[settings]
+line_length=80
```
Comment (danni) [inline .isort.cfg]: 78

Comment (timheap) [inline .isort.cfg]: Should I also add pep8 / flake8 configs for this?

Comment (danni) [inline .isort.cfg]: I thought that was pep8.

Comment (timheap) [inline .isort.cfg]: The config is for isort. pep8 suggests lines of 79 chars.

Comment (danni) [inline .isort.cfg]: Okay, never mind.

**Pull request merged**